templates/core/terraform/variables.tf

```diff
@@ -12,6 +12,12 @@ variable "location" {
   description = "Azure region for deployment of core TRE services"
 }
 
+variable "deploy_ui" {
```

Comment:
Are you using this var in Terraform and I've missed it?

Reply:
I was, but not now - good spot!